src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -542,8 +542,7 @@ protected DynamoModel(IStartConfiguration config)
             geometryFactoryPath = config.GeometryFactoryPath;
 
             IPreferences preferences = CreateOrLoadPreferences(config.Preferences);
-            var settings = preferences as PreferenceSettings;
-            if (settings != null)
+            if (preferences is PreferenceSettings settings)
```

Comment:
I think this declaration pattern already checks for null value.
From msdn:
`Beginning with C# 7.0, a declaration pattern with type T matches an expression when an expression result is non-null and any of the following conditions are true:...`
Example:
https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/functional/pattern-matching#null-checks